modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/model/FDSActionDropdownItem.java

```diff
@@ -14,6 +14,17 @@
  */
 public class FDSActionDropdownItem extends DropdownItem {
 
+	public FDSActionDropdownItem(
+		String confirmationMessage, Boolean highlighted, String href,
+		String icon, String id, String label, String method,
+		String permissionKey, String target) {
```

Comment:
When making an overload with a new argument, it's typically best if it's done on the biggest overload, that contains all settable arguments. That way we end up with new comprehensive overload. In this case, it would be the last one in file, with 16 arguments. Sometimes, we can add a simpler overload _in addition_, if there is a common use case for it, but I don't this there is in this case. Sometimes, we can add a simpler overload if we know arguments are mutually exclusive, but I don't thing they are in this case. For example, a dev may want a highlighted item with a visibility filter.